### PR TITLE
Add desktop version

### PR DESCRIPTION
**Pull request by Samuellekpor**
1. Add desktop images to HTML.
2. Update the CSS file with media queries for the desktop version.
3. Update HTML file with tags for desktop.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -14,134 +14,160 @@
...
+					<li>javaScript</li>
+				</ul>
+				<button class="button" type="button">See Project</button>
 			</div>
-			<p class="section-description">A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
-			<ul class="technologies">
-				<li>html</li>
-				<li>css</li>
-				<li>javaScript</li>
-			</ul>
```
Comment (Kamba56) [inline index.html:76]: - [ ] Kindly use an indentation of 2 spaces in all your document for readability. This is [best practice](https://github.com/microverseinc/curriculum-html-css/blob/main/articles/html_css_best_practices.md) and a convention.

**Review by Kamba56 — CHANGES_REQUESTED** [review]
Hi @Samuellekpor, 

Kudos to you. You have done a great job building your portfolio. Keep up the good work and the sky will be your starting point. However, there are some changes I will suggest you look into. 

## Highlights
- [x] All linters passed
- [x] Correct use of GitHub flow
- [x] Descriptive PR title and description
- [x] Good job implementing the UI

## Required Changes ♻️
_Please check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by mgmediaweb — CHANGES_REQUESTED** [review]
Hi @Samuellekpor , 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.

almost1

 ## Required Changes ♻️

The previous reviewer kindly suggested that you change the indentation of your HTML/CSS files to just 2 spaces, I advise you to make this change before submitting another review.

_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html (hunk 1 above)
Comment (Samuellekpor) [reply index.html:76]: Dear @mgmediaweb and @Kamba56 
Please can I know on which line is the issue?
In my code, I have 2 spaces indentations everywhere as I corrected the 4 lines. Please help me solve it before I run out of reviews. Thanks

**Comment by Samuellekpor** [comment]
Dear @mgmediaweb  and @Kamba56
Please can I know on which line is the issue?
In my code, I have 2 spaces indentations everywhere as I corrected the 4 lines. Please help me solve it before I run out of reviews. Thanks

File: index.html (hunk 1 above)
Comment (mgmediaweb) [reply index.html:76]: Kindly, all over your site within Github there is excessive indentation
image

In this example I show you how it should look
image

Comment (Samuellekpor) [reply index.html:76]: @mgmediaweb 
Is there a way to change that in the GitHub code, please? This is what it looks like in my code in VSCode.

code-capture

Comment (Kamba56) [reply index.html:76]: Please save it and push the changes to your GitHub.

**Review by Munsa1 — APPROVED** [review]
Hi @Samuellekpor, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights
:heavy_check_mark: Descriptive Readme
:heavy_check_mark: Passing Linters
:heavy_check_mark: Correct Gitflow

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**